n3fit/src/n3fit/hyper_optimization/mongofiletrials.py

```diff
@@ -154,6 +154,8 @@ def __init__(
             f"mongo://{self.db_host}:{self.db_port}/{self._process_db_name(self.db_name)}/jobs"
         )
         self.workers = []
+        self.log_files = []
+        self.output_path = replica_path.parts[-3]
```

Comment:
Why specifically at this location?

Reply:
I define them together since these are related attributes used to define the workers list and corresponding output files:
https://github.com/NNPDF/nnpdf/blob/4974bda5ec12dce1535b017fe6e06a993b2407b8/n3fit/src/n3fit/hyper_optimization/mongofiletrials.py#L270-L277

Reply:
Sorry, my question wasn't clear. I was asking why the logs are saved in that particular path? Given that usually this will be run with more than one worker, why not collect them in some folder, for example?

Reply:
Oh. Sorry. Now I see. `self.output_path ` is not a path but the name of the runcard or output name given by the user. So, to avoid this confusion I changed the name of this variable to `self.job_output_name`. Remember that `replica_path` is something like `path/RUNCARD_NAME/nnfit/replica_N`. So, `RUNCARD_NAME = replica_path.parts[-3]`.